pom.xml

```diff
@@ -383,6 +380,28 @@ under the License.
 	</dependencyManagement>
 
 	<profiles>
+		<profile>
+			<id>opensearch1</id>
+			<activation>
+				<jdk>[1.8,)</jdk>
```

Comment:
do we still need 1.8 here?

Flink 2.0 requires at least 11...

Reply:
Yeah we build against 1.20 as well

Reply:
true